tools/chplcheck/src/config.py

```diff
@@ -79,6 +126,13 @@ def add_arguments(cls, parser: argparse.ArgumentParser, prefix: str = ""):
             default=[],
             help="Add a custom rule file for chplcheck",
         )
+        parser.add_argument(
+            f"--{prefix}setting",
+            action=RuleSettingAction,
+            default=dict(),
```

Comment:
Since the action auto-sets a default if one is missing, you don't need this line, do you?

Reply:
correct, its just more explicit and I'd prefer to keep it